include/roboteam_ai/bt/BehaviorTree.hpp

```diff
@@ -9,11 +9,13 @@ namespace bt {
 class BehaviorTree : public Node {
         FRIEND_TEST(BehaviorTreeTest, it_sets_blackboards);
     public:
-
+        std::string name;
```

Comment:
Don't we usually return the name from node_name as a literal? Allowing it to get inlined?